.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# All repo
+* @turbaszek
```

Comment:
I did not added all committers because not everyone may want to get updates. Let me know if I should add you

Reply:
I'm happy to manage the items you put me down for :-)

Reply:
I will do my best to support. Please add me as well, Tomek.